tests/test_wave_03.py

```diff
@@ -59,7 +59,7 @@ def test_friends_unique_movies_not_duplicated():
     # ****** Add assertions here to test that the correct movies are in friends_unique_movies **********
     # **************************************************************************************************
 
-@pytest.mark.skip()
+# @pytest.mark.skip()
```

Comment:
Hi Poppy! You're also missing the assert to check if the correct movies are in `friends_unique_movies`.